staff/albert-escolar/spotify-template/spotify-desktop-template/style.sass

```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat')
+
+$spotify-black: #181818
```

Comment:
@Escoly nice use of variables! You could also create a variables sass file and import it in all sass files needed.